packages/core/src/browser/shell/sidebar-menu-widget.tsx

```diff
@@ -55,14 +55,14 @@ export class SidebarBottomMenuWidget extends ReactWidget {
   }
 
   removeMenu(menuId: string): void {
-      const menu = this.menus.find(m => m.id === menuId);
-      if (menu) {
-          const index = this.menus.indexOf(menu);
-          if (index !== -1) {
-              this.menus.splice(index, 1);
-              this.update();
-          }
+    const menu = this.menus.find(m => m.id === menuId);
+    if (menu) {
+      const index = this.menus.indexOf(menu);
+      if (index !== -1) {
+        this.menus.splice(index, 1);
+        this.update();
       }
+    }
   }
 
   protected onClick(e: React.MouseEvent<HTMLElement, MouseEvent>, menuPath: MenuPath): void {
```

Comment:
Minor, and related only to old code: this method uses the click as the anchor for the menu. In VSCode, the menu is always anchored to the right of the element, so it likely uses the element's coordinates instead.

Reply:
Reference for VSCode open menu code: https://github.com/microsoft/vscode/blob/99f71a032ea932670744d14d6364903a00b99ae3/src/vs/base/browser/ui/menu/menubar.ts#L957